src/infobubble.js

```diff
@@ -1104,40 +1116,44 @@ InfoBubble.prototype.panToView = function() {
     return;
   }
 
-  var anchorHeight = this.getAnchorHeight_();
-  var height = this.bubble_.offsetHeight + anchorHeight;
-  var map = this.get('map');
-  var mapDiv = map.getDiv();
-  var mapHeight = mapDiv.offsetHeight;
+  var autopanMargin = this.get('autopanMargin');										// get the edge margin from options (or default)
```

Comment:
ok no worries - comments shifted to line above and line length limited to 80 char.